testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -1413,8 +1412,7 @@ public void testAcidUpdateWithSubqueryAssignment()
 
                 // UPDATE while reading from another transactional table. Multiple transactional could interfere with ConnectorMetadata.beginQuery
                 onTrino().executeQuery(format("UPDATE %s SET column2 = (SELECT min(name) FROM %s)", tableName, secondTable));
-                // TODO (https://github.com/trinodb/trino/issues/8268) verifySelectForTrinoAndHive("SELECT * FROM " + tableName, "true", row(1, "AFRICA"), row(2, "AFRICA"));
-                verifySelect("onTrino", onTrino(), "SELECT * FROM " + tableName, "true", row(1, "AFRICA"), row(2, "AFRICA"));
+                verifySelectForTrinoAndHive("SELECT * FROM " + tableName, "true", row(1, "AFRICA"), row(2, "AFRICA"));
```

Comment:
I like the fact we run a test on hive now too.
However, we're just masking a bug here. https://github.com/trinodb/trino/issues/8268 says something about 

> repro steps in #8267 as a TODO

but the code here is changing (and TODO being removed)

we need to make sure than anyone doing that issue knows how to reproduce the problem

Reply:
But haven't we added the TODO steps here 
`verifySelectForTrinoAndHive("SELECT * FROM " + tableName, "true", row(1, "AFRICA"), row(2, "AFRICA"));`

Am I missing any other steps here ?

Reply:
well, but you're removing it, right?

Reply:
Yeah !! But if it closes the issue.. do we still need it in the comment ?

Reply:
Have added an additional step to todo.